plugin/advisor/catalog/catalog.go

```diff
@@ -26,6 +27,11 @@ type IndexFind struct {
 	IndexName string
 }
 
+// PKFind is the API message for find primary key
+type PKFind struct {
```

Comment:
Prefer to use full name for API. PrimaryKeyFind.

Reply:
Need I change the "FindPK" to "FindPrimaryKey" also?

Reply:
Yeah, s/PK/PrimaryKey.